Assets/BossRoom/Scripts/Shared/Game/Action/ActionUtils.cs

```diff
@@ -88,7 +88,7 @@ public static bool IsValidTarget(ulong targetId)
             //note that we DON'T check if you're an ally. It's perfectly valid to target friends,
             //because there are friendly skills, such as Heal.
 
-            if (!NetworkManager.Singleton.SpawnManager.SpawnedObjects.TryGetValue(targetId, out var targetChar))
+            if (NetworkManager.Singleton.SpawnManager == null || !NetworkManager.Singleton.SpawnManager.SpawnedObjects.TryGetValue(targetId, out var targetChar))
```

Comment:
Is this related?

Reply:
Oops I forgot to document this change. It is not related, just a bug that happens when disconnecting while in-game. It did not happen often earlier (since the only case where this happened was when a host quit the game while clients are still playing), but when testing with the disconnect button it caused lots of exceptions.